contrib/README.md

```diff
@@ -0,0 +1,33 @@
+Julia Config directory README  
+=============================  
+  
+ackrc - config for Ack  
+BBEditTextWrangler-julia.plist - language module for BBEdit and TextWrangler text editors  
+build_executable.jl - Support building standalone executables  
+build_sysimg.jl - Build a system image binary  
+check-whitespace.sh - Check for trailing white space  
+ctags - Add more keywords to ctags regex  
+debug_bootstrap.gdb - bootstrap process using the debug build  
+filterArgs.sh - Update library search code to use only tokens that start with "-L"  
+filterDupLibs.sh - script to filter out duplicate libraries  
+fixup-libgfortran.sh - Include libgfortran and libquadmath for installations  
+install.sh - installation script with different permissions  
+install-tmbundle - TextMate bundle install script  
+julia.appdata.xml - Appdata config file  
+julia.desktop - GNOME desktop config file  
+julia.hrc - Julia syntax highlighting definition for Colorer   
+julia.lang - GtkSourceView (GTK+ framework) for multiline text editing config file  
+julia-mode.el - Emacs Julia mode config file  
+Julia_Notepad++.xml - Notepad++ config file  
+julia.svg - julia svg image file  
+Julia.tmbundle/ - TextMate support files  
+julia.xml - KDE editors Kate and Kwrite config file  
+mac/ - Mac install files  
+Notepad++_2_Julia.ahk - Auto Hotkey for Notepad++ config file  
+README.ackrc.txt - README for ackrc config file  
+README.ctags.txt - README for ctags config file  
+relative_path.sh - convert absolute path into relative path script  
+repackage_system_suitesparse4.make - reinstall_suitesparse4 script  
+stringreplace.c - Cygwin helper script  
```

Comment:
This isn't for Cygwin, we use it everywhere during `make install`. It's to replace strings to hardcoded paths in binaries. It just has a limited history with the current file name, it used to be called stringpatch but Windows does funny things with admin rights and 32-bit executables that have "patch" or "install" in their names. Here's the history of the file at its old name: https://github.com/JuliaLang/julia/commits/b42d5807cfbae15e520dcf23c3eff32addb3e437/contrib/stringpatch.c